mne/commands/tests/test_commands.py

```diff
@@ -202,4 +203,37 @@ def test_watershed_bem():
         mne_watershed_bem.run()
 
 
+@slow_test
+@requires_mne
+@requires_freesurfer
+@sample.requires_sample_data
+def test_flash_bem():
```

Comment:
Does this test work for you? On my system I get:
```
$ nosetests mne/commands/tests/test_commands.py:test_flash_bem --verbose
======================================================================
ERROR: Test mne flash_bem
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/larsoner/custombuilds/mne-python/mne/utils.py", line 623, in dec
    return function(*args, **kwargs)
  File "/home/larsoner/custombuilds/mne-python/mne/utils.py", line 623, in dec
    return function(*args, **kwargs)
  File "/home/larsoner/.local/lib/python2.7/site-packages/numpy/testing/decorators.py", line 146, in skipper_func
    return f(*args, **kwargs)
  File "/home/larsoner/custombuilds/mne-python/mne/commands/tests/test_commands.py", line 235, in test_flash_bem
    mne_flash_bem.run()
  File "/home/larsoner/custombuilds/mne-python/mne/commands/mne_flash_bem.py", line 81, in run
    show=show)
  File "/home/larsoner/custombuilds/mne-python/mne/bem.py", line 1494, in make_flash_bem
    run_subprocess(cmd, env=env, stdout=sys.stdout)
  File "<string>", line 2, in run_subprocess
  File "/home/larsoner/custombuilds/mne-python/mne/utils.py", line 540, in verbose
    return function(*args, **kwargs)
  File "/home/larsoner/custombuilds/mne-python/mne/utils.py", line 800, in run_subprocess
    p = subprocess.Popen(command, *args, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 702, in __init__
    errread, errwrite), to_close = self._get_handles(stdin, stdout, stderr)
  File "/usr/lib/python2.7/subprocess.py", line 1136, in _get_handles
    c2pwrite = stdout.fileno()
AttributeError: StringIO instance has no attribute 'fileno'
-------------------- >> begin captured stdout << ---------------------
Command not found: mri_ms_fitparms
```
So there is some problem with `fileno`. It's also strange that it says the `mri_ms_fitparms` command is not found, since I do have it.